docs/api.rst

```diff
@@ -110,3 +110,60 @@ Methods
     Dataset.temporal.center_times
 
 .. _dsmeth_1:
+
+.. currentmodule:: xcdat
+
+.. _cdat-mapping:
+
+CDAT Mapping Table
+------------------
+
+The table below maps the supported xCDAT operations to the equivalent CDAT and xCDAT APIs.
+It is especially useful for those who are transitioning over from CDAT to xarray/xCDAT.
+
+.. list-table::
+   :header-rows: 1
+   :widths: 15 10 30
+
+   * - How do I...
```

Comment:
@tomvothecoder this is great! Can we also get a couple of the other coords relevant mappings from #200 added below

Reply:
Hi @durack1, glad to hear! That sounds useful, I'll make sure to add it (the API calls will be to `cf_xarray` but that's included in `xcdat`).